src/zenml/cli/stack.py

```diff
@@ -681,3 +688,125 @@ def down_stack(force: bool = False) -> None:
             f"Suspending resources for active stack '{stack_.name}'."
         )
         stack_.suspend()
+
+
+@stack.command("export")
+@click.argument("filename", type=str, required=False)
+@click.argument("stack_name", type=str, required=False)
+def export_stack(filename: Optional[str], stack_name: Optional[str]) -> None:
+    """Export a stack to YAML."""
+
+    # Get configuration of given stack
+    # TODO: code duplicate with describe_stack()
+    repo = Repository()
+
+    active_stack_name = repo.active_stack_name
+    stack_name = stack_name or active_stack_name
+
+    if not stack_name:
+        cli_utils.warning("No stack is set as active!")
+        return
+
+    stack_configurations = repo.stack_configurations
+    if len(stack_configurations) == 0:
+        cli_utils.warning("No stacks registered!")
+        return
+
+    try:
+        stack_configuration = stack_configurations[stack_name]
+    except KeyError:
+        cli_utils.error(f"Stack '{stack_name}' does not exist.")
+        return
+
+    # create a dict of all components in the specified stack
+    component_data = {}
+    for component_type, component_name in stack_configuration.items():
+        component = repo.get_stack_component(
+            component_type, name=component_name
+        )
+        component_dict = {
+            key: value
+            for key, value in json.loads(component.json()).items()
+            if key != "uuid" and value is not None
+        }
+        component_dict["flavor"] = component.FLAVOR
+        component_data[str(component_type)] = component_dict
+
+    # write zenml version and stack dict to YAML
+    yaml_data = {
+        "zenml_version": zenml.__version__,
+        "stack_name": stack_name,
+        "components": component_data,
+    }
+    if filename is None:
+        filename = stack_name + ".yaml"
+    write_yaml(filename, yaml_data)
+    cli_utils.declare(f"Exported stack '{stack_name}' to file '{filename}'.")
+
+
+@stack.command("import")
+@click.argument("filename", type=str, required=True)
+@click.argument("stack_name", type=str, required=False)
+@click.pass_context
+def import_stack(
+    ctx: click.Context, filename: str, stack_name: Optional[str]
+) -> None:
+    """Import a stack from YAML."""
+    data = read_yaml(filename)
+
+    # assert zenml version is the same
+    if data["zenml_version"] != zenml.__version__:
+        cli_utils.error(
+            f"Cannot import stacks from other ZenML versions. "
+            f"The stack was created using ZenML version {data['zenml_version']}, "
+            f"you have version {zenml.__version__} installed."
+        )
+        return
```

Comment:
Same as @strickvl s comment above, the return can be removed as `cli_utils.error` is raising an exception

Reply:
Yep, just removed it, thanks!